package-lock.json

```diff
@@ -1,4908 +1,8 @@
 {
   "name": "@rsksmart/rsk-precompiled-abis",
-  "version": "4.0.2-IRIS",
-  "lockfileVersion": 2,
```

Comment:
Check your version of node. It should be v16

Reply:
Done